tests/memarray/main.c

```diff
@@ -101,7 +101,7 @@ void free_memory(struct block_t *head)
 int main(void)
 {
     printf("MAX_NUMBER_BLOCKS: %d\n", MAX_NUMBER_BLOCKS);
-    printf("NUMBER_OF_TESTS: %d\n", NUMBER_OF_TESTS);
+    printf("NUMBER_OF_TESTS: %d\n", 2 * NUMBER_OF_TESTS);
```

Comment:
This change isn't immediately clear to me. Please add a comment (or change the definition of `NUMBER_OF_TESTS`).

Reply:
I'll add a comment. But I basically added a second test loop, so it now runs twice `NUMBER_OF_TESTS` tests.

Reply:
Mhh... but than `NUMBER_OF_TESTS` should be whatever it was before multiplied by 2, shouldn't it? Otherwise, this is very confusing naming ;-).